db/seeds.rb

```diff
@@ -10,43 +10,15 @@
 require 'ffaker'
 expirations = ((Date.today - 7.days)..(Date.today + 6.months)).to_a
 
-10.times do
-  Passenger.create! name: FFaker::Name.name,
-                    address: FFaker::Address.street_address,
-                    email: FFaker::Internet.email,
-                    phone: FFaker::PhoneNumber.short_phone_number,
-                    wheelchair: false,
-                    active: true,
-                    permanent: false,
-                    expiration: expirations.sample
-  Passenger.create! name: FFaker::Name.name,
-                    address: FFaker::Address.street_address,
-                    email: FFaker::Internet.email,
-                    phone: FFaker::PhoneNumber.short_phone_number,
-                    wheelchair: false,
-                    active: true,
-                    permanent: true
+20.times do
```

Comment:
So you want 100 passengers whose registrations, in equal frequency:

1. Don't have an expiration
2. Expire in 2 days
3. Expired 2 days ago
4. Expired 4 days ago
5. Might've expired anytime in the last week, or might expire anytime in the next 6 months.

Am I reading that correctly?